### ISSUE TITLE
pypy no longer installed in trusty image

### ISSUE THREAD

**Issue by asottile** [travis-ci/travis-ci#8081]
Today: https://travis-ci.org/pre-commit/pre-commit/jobs/253037772
Three days ago: https://travis-ci.org/pre-commit/pre-commit/jobs/251816068#L6

No mention of pypy in todays update however: https://docs.travis-ci.com/user/build-environment-updates/2017-07-12

**Comment by BanzaiMan** [comment]
Closing as a duplicate of #6865.

**Comment by asottile** [comment]
@BanzaiMan this is not a duplciate of #6865, `pypy` **was installed in trusty 3 days ago**

**Comment by BanzaiMan** [comment]
The issue is that the new image does not have PyPy pre-installed, right?

**Comment by asottile** [comment]
Correct, whereas three days ago the trusty image did have it pre-installed.

**Comment by BanzaiMan** [comment]
Sorry for the delay. Could you point to a build that shows PyPy being pre-installed on a Trusty image?

Thanks.

**Comment by BanzaiMan** [comment]
Oops. You had that in the description. Sorry for the noise.

**Comment by BanzaiMan** [comment]
We will continue with #8160.

**Comment by Lyoness** [comment]
The issue is:  This build got re-routed to a _different image_ than intended though the config remained unchanged.   Still de-bugging the reason.  As a workaround @asottile :  can you try specifying the following in your .travis.yml: 

`language: __sugilite__`

This is the build image stack that has the pypy preinstalled.  I've sent you a [PR](https://github.com/pre-commit/pre-commit/pull/567) accordingly.

**Comment by asottile** [comment]
That "language" does indeed work, but I consider it a less expressive workaround than `python: pypy-5.7.1`: https://github.com/pre-commit/pre-commit/commit/5c0783b2d0668cc9751735e0f7eea1e2f79c1448

**Comment by asottile** [comment]
As you can see, I've hit this quite a few times today!  Looking forward to a fix :)
